pymc3/tests/test_distributions.py

```diff
@@ -1989,32 +1988,32 @@ def test_kroneckernormal(self, n, m, sigma):
             scipy_args=evd_args,
         )
 
-        dom = Domain([np.random.randn(2, N) * 0.1], edges=(None, None), shape=(2, N))
-
-        self.check_logp(
-            KroneckerNormal,
-            dom,
-            std_args,
-            kron_normal_logpdf_cov,
-            extra_args=cov_args,
-            scipy_args=cov_args,
-        )
-        self.check_logp(
-            KroneckerNormal,
-            dom,
-            std_args,
-            kron_normal_logpdf_chol,
-            extra_args=chol_args,
-            scipy_args=chol_args,
-        )
-        self.check_logp(
-            KroneckerNormal,
-            dom,
-            std_args,
-            kron_normal_logpdf_evd,
-            extra_args=evd_args,
-            scipy_args=evd_args,
-        )
+        # dom = Domain([np.random.randn(2, N) * 0.1], edges=(None, None), shape=(2, N))
```

Comment:
This is again related to how `logp` expects the shape of `value` variable to be. Does it derive it using `params` ? If so it'll require override similar to `_shape_from_params` but for `logp`

For instance in these tests it expects the shape of `value` to be 1 dimensional. But receives a two dimensional array that we provide from here.